libs/labelbox/tests/integration/test_model_config.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+def test_create_model_config(client, valid_model_id):
+    model_config = client.create_model_config("model_config", valid_model_id, {"param": "value"})
```

Comment:
I don't see a cleanup of this model config, we should make sure it's deleted after the test case runs